colossalai/pipeline/stage_manager.py

```diff
@@ -18,7 +18,9 @@ class PipelineStageManager:
         stage (int): The current stage.
     """
 
-    def __init__(self, pg_mesh: ProcessGroupMesh, pipeline_axis: int, is_virtual: bool = False) -> None:
+    def __init__(
+        self, pg_mesh: ProcessGroupMesh, pipeline_axis: int, is_virtual: bool = False, num_model_chunks=1
```

Comment:
It seems that `is_virtual` is not used.

Reply:
it works together with a previous commit https://github.com/hpcaitech/ColossalAI/pull/4448/files#
specifically https://github.com/hpcaitech/ColossalAI/blob/8aed02b957082e676cf424068583c78240b27f9d/colossalai/pipeline/stage_manager.py#L44

Reply:
Current code does not use it. Just delete it.

Reply:
retained is_virtual as discussed